### PR TITLE
Iccv 2023

### PR DESCRIPTION
**Pull request by robintrmbtt**

### CODE REVIEW

File: collections/_posts/2023-10-06-ICCV_2023.md [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+layout: review
+title: "ICCV 2023 - Selection of papers"
+tags: iccv-2023
+author: "Robin Trombetta, Matthis Manthe"
+---
+
+
+# Note
+
+The purpose of this post is to compile a number of papers that have been presented at 2023 International Conference on Computer Vision (ICCV) and which we feel may be of interest for people working in deep learning and/or medical imaging.
```
Comment (nathanpainchaud) [inline collections/_posts/2023-10-06-ICCV_2023.md:11]: Missing word:
```suggestion
The purpose of this post is to compile a number of papers that have been presented at the 2023 International Conference on Computer Vision (ICCV) and which we feel may be of interest for people working in deep learning and/or medical imaging. 
```

File: collections/_posts/2023-10-06-ICCV_2023.md [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+
+<div style="text-align:center">
+<img src="/collections/images/iccv_2023/preserving_modality.jpg" width=800></div>
+
+S. Swetha, M. Nayeem Rizve, N. Shvetsova, H. Kuehne, M. Shah, **Preserving Modality Structure Improves Multi-Modal Learning**, Proceedings of the IEEE/CVF International Conference on Computer Vision (ICCV), 2023.
+
+[Link to the article](https://openaccess.thecvf.com/content/ICCV2023/papers/Swetha_Preserving_Modality_Structure_Improves_Multi-Modal_Learning_ICCV_2023_paper.pdf)
+
+> "In this context, we propose a novel Semantic-Structure-Preserving Consistency approach to improve generalizability by preserving the modality-specific relationships in the joint embedding space. To capture modality-specific semantic relationships between samples, we propose to learn multiple anchors and represent the multifaceted relationship between samples with respect to their relationship with these anchors. To assign multiple anchors to each sample, we propose a novel Multi-Assignment Sinkhorn-Knopp algorithm."
+
```
Comment (nathanpainchaud) [inline collections/_posts/2023-10-06-ICCV_2023.md:96]: Missing new line compared to all the other listed papers:
```suggestion

&nbsp;

```

**Review by nathanpainchaud — CHANGES_REQUESTED** [review]
You did an amazing job compiling all these papers! A great many thanks to you for doing all this work for us :wink: 

Just 2 minor nitpicks to fix a typo + formatting issue I noticed. Also just letting you know I took the liberty to rename the mentions of _articles_ to _papers_ since it's the more appropriate term in English (but I didn't want to leave those as comments since there were a lot of occurences of the term :sweat_smile:)

**Comment by nathanpainchaud** [comment]
@robintrmbtt A question I forgot to ask in my review: for the thematic subsections in the _other papers_ section, did you come up with those categories yourselves, or was it how the papers were presented at the conference itself (e.g. the themes of the poster sessions?)

**Comment by robintrmbtt** [comment]
Thank you for the review! For the thematic subsections, there were mainly directly taken from the themes of the sessions at the conference, with few merging/re-assignement that we thought were judicious

**Review by nathanpainchaud — APPROVED** [review]

**Pull request merged**